social-network/src/main/java/com/levi9/socialnetwork/Service/impl/GroupServiceImpl.java

```diff
@@ -46,4 +53,18 @@ public Group deleteGroup(Long groupId) throws ResourceNotFoundException{
 		return group;
 	}
 	
+	public User addUserToGroup(RequestDTO requestDTO) throws ResourceNotFoundException, ResourceExistsException {
+		Group group = groupRepository.findById(requestDTO.getIdGroup()).orElseThrow(() -> new ResourceNotFoundException("Group is not found for this id ::" + requestDTO.getIdGroup()));
```

Comment:
You can extract this message into a String constant. That way if a change is needed, you will only change it once.